samples/dfu/README.rst

```diff
@@ -7,44 +7,70 @@ Overview
 This DFU application demonstrates how to connect with Golioth and use Device
 Firmware Upgrade (DFU) procedure.
 
+The app is first built and flashed to the device which will then listen to
+Golioth Cloud for a new firmware binary to become available. A second binary is
+built with a new version number, registered as an artifact, and depolyed as a
+release. The device will detect this change and automatically download and
+install.
+
 Requirements
 ************
 
 - Golioth credentials
 - Network connectivity
 
-Using nRF9160 Feather with nRF Connect SDK
-******************************************
+Using nRF9160 Feather or Thingy:91 with nRF Connect SDK
+*******************************************************
+
+A full nRF9160 firmware `flashing overview`_ is available in the Golioth docs.
+Please use that resource to install the tools necessary to flash these devices.
+
+Configure the following Kconfig options based on your Golioth credentials:
+
+- GOLIOTH_SYSTEM_CLIENT_PSK_ID  - PSK ID of registered device
+- GOLIOTH_SYSTEM_CLIENT_PSK     - PSK of registered device
 
-Build Zephyr sample application for nRF9160 Feather:
+by adding these lines to configuration file (e.g. ``prj.conf``):
+
+.. code-block:: cfg
+
+   CONFIG_GOLIOTH_SYSTEM_CLIENT_PSK_ID="my-psk-id"
+   CONFIG_GOLIOTH_SYSTEM_CLIENT_PSK="my-psk"
+
+Build Zephyr sample application:
```

Comment:
This sounds like a generic doc improvement, so should be in separate commit. Hard to say if it is valid in current form, as there are too many changes as part of single commit.